### PR TITLE
doc: Switch screenshot to use Chrome

### PR DESCRIPTION
**Pull request by nhooyr**
Safari runs code-server horribly. We shouldn't even pretend that it sort
of works.

See the issues tagged browser-safari!

### CODE REVIEW

**Comment by nhooyr** [comment]
You can see the difference live here: https://github.com/cdr/code-server/tree/ss-20c2

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1 @@
-* @code-asher @nhooyr
```
Comment (oxy) [inline .github/CODEOWNERS:1]: Not sure if removing CODEOWNERS here was intentional!

**Review by oxy — COMMENTED** [review]
Screenshot looks good, not sure what happened with `.github/CODEOWNERS`

File: .github/CODEOWNERS (hunk 1 above)
Comment (nhooyr) [reply .github/CODEOWNERS:1]: It was! We strive to break down each PR into individual commits. Makes it much easier to review. See the justification in the second commit.

Creating a new PR for every little change tends to get unwieldy.

Comment (nhooyr) [reply .github/CODEOWNERS:1]: But yea know what actually, I'll try to do that from now on. It's def confusing to mesh things together in unrelated PRs.

**Pull request merged**